ktlint-ruleset-standard/src/main/kotlin/com/pinterest/ktlint/ruleset/standard/TrailingCommaRule.kt

```diff
@@ -192,6 +193,34 @@ public class TrailingCommaRule :
         node.reportAndCorrectTrailingCommaNodeBefore(inspectNode, emit, autoCorrect)
     }
 
+    private fun visitClassEntry(
+        node: ASTNode,
+        emit: (offset: Int, errorMessage: String, canBeAutoCorrected: Boolean) -> Unit,
+        autoCorrect: Boolean
+    ) {
+        val psi = node.psi
+        require(psi is KtClass)
+
+        if (psi.isEnum()) {
+            val classBody = node.children().singleOrNull { it.psi is KtClassBody }
+                ?: return // Nothing to do for empty enum class
+
+            val lastRelevantChild = classBody.psi.children.toList().asReversed()
+                .first()
+
+            val fullAST = psi
+            val fullPsi = classBody.psi
```

Comment:
Those variables seem not to be used. Please cleanup.

Reply:
Sorry, I didn't anticipate a full code review on this. The `WIP` in the commit message is short for work-in-progress. I'm having some issues with the AST not containing the semicolon element. These variables were left for debugging, which is not completed yet.

Reply:
Yeah, I know the WIP abbreviation. I am not sure whether the PR had "draft" status at the time I reviewed the code. But anyways, regard it as early feedback ;-)